Sources/MoonlightTests/MoonlightTestCase.swift

```diff
@@ -51,6 +52,7 @@ public extension MoonlightTestCase {
                 }
                 .eraseToAnyPublisher()
             }
+            .last()
```

Comment:
Would it not have made more sense to make only 1 function that takes array of events and then create a function for single event that calls function with many events as [event]. Now we have to support two functions, which seemingly do same things, although one is a bit easier to read.